src/Platforms/SqlitePlatform.php

```diff
@@ -718,22 +718,7 @@ protected function getReservedKeywordsClass()
      */
     protected function getPreAlterTableIndexForeignKeySQL(TableDiff $diff)
     {
-        if (! $diff->fromTable instanceof Table) {
-            throw new Exception(
-                'Sqlite platform requires for alter table the table diff with reference to original table schema'
-            );
-        }
-
-        $sql = [];
-        foreach ($diff->fromTable->getIndexes() as $index) {
-            if ($index->isPrimary()) {
-                continue;
-            }
-
-            $sql[] = $this->getDropIndexSQL($index, $diff->name);
-        }
-
-        return $sql;
+        return [];
```

Comment:
`DROP INDEX` was added in https://github.com/doctrine/dbal/commit/38bd06935a6e596a61a15cdf34f576d9a952ac09#diff-58c437cdf1e89c30ab050c8e80da4ac0fe8364076b0707ce0b0d71080eca7f0fR504 in 2012 when FK support for Sqlite was introduced by @hason